client/components/payment-method-capability-status-pill/__tests__/payment-method-capability-status-pill.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import PaymentMethodCapabilityStatusPill from '..';
+import { useGetCapabilities } from 'wcstripe/data/account';
+import UpeToggleContext from 'wcstripe/settings/upe-toggle/context';
+
+jest.mock( 'wcstripe/data/account', () => ( {
+	useGetCapabilities: jest.fn(),
+} ) );
+
+describe( 'PaymentMethodCapabilityStatusPill', () => {
+	beforeEach( () => {
+		useGetCapabilities.mockReturnValue( {} );
+	} );
+
+	it( 'should render the "Pending activation" text', () => {
+		useGetCapabilities.mockReturnValue( {
+			giropay_payments: 'pending',
+			card_payments: 'active',
+		} );
+		render(
+			<UpeToggleContext.Provider value={ { isUpeEnabled: true } }>
+				<PaymentMethodCapabilityStatusPill
+					id="giropay"
+					label="giropay"
+				/>
+			</UpeToggleContext.Provider>
+		);
+
+		expect(
+			screen.queryByText( 'Pending activation' )
+		).toBeInTheDocument();
+	} );
+
+	it( 'should not render when UPE is disabled', () => {
+		useGetCapabilities.mockReturnValue( {
+			giropay_payments: 'pending',
+			card_payments: 'pending',
+		} );
+		const { container } = render(
+			<UpeToggleContext.Provider value={ { isUpeEnabled: false } }>
+				<PaymentMethodCapabilityStatusPill
+					id="giropay"
+					label="giropay"
+				/>
+			</UpeToggleContext.Provider>
+		);
+
+		expect( container.firstChild ).toBeNull();
+	} );
+
+	it( 'should not render when the capability is "active"', () => {
```

Comment:
I was actually thinking of adding to this test so that it tests `active`, `inactive` and `unrequested` instead of just `active`. In case `capabilityStatus === 'pending'` is updated to `capabilityStatus === 'pending' || ...`. Then I thought that may be too much since it already tested both paths.

Reply:
> Are you recommending removing this test case?

Absolutely not. :D